src/Schema/SqliteSchemaManager.php

```diff
@@ -582,7 +582,7 @@ private function getCreateTableSQL(string $table): string
 AND name = ?
 SQL
             ,
-            [$table],
+            [trim($table, '"')], // Unquote the identifier
```

Comment:
I'm not sure about this change, see discussion in https://github.com/doctrine/dbal/pull/6765#discussion_r1941396459

Basically during our test we are calling `$this->schemaManager->introspectTable('"user"');`, but here the matching should not be done using the extra quotes.

I do not know if there are valid table names that can have leading / trailing quotes in SQLite.

Note that user does not seem to be a reserved keyword in SQLite (https://www.sqlite.org/lang_keywords.html) but I tried with `ABORT` and it worked correctly with the name trimmed.